routes/api.php

```diff
@@ -13,4 +15,11 @@
     Route::delete('/{city}', [CityController::class, 'destroy'])->name('cities.destroy');
     Route::put('/{city}', [CityController::class, 'update'])->name('cities.update');
     Route::get('/', [CityController::class, 'getCities']);
+});
+
+Route::group(['prefix' => 'airlines'], function () {
```

Comment:
```suggestion
Route::group(['prefix' => 'airlines', 'controller' => AirlineController::class], function () {
```
Adding the grouping by controller will allow you to:
```
// Do this
Route::get('/', 'getAirlines');

// Instead of this
Route::get('/', [AirlineController::class, 'getAirlines'])
```